src/05-derivatives/02-common-data-types.md

```diff
@@ -0,0 +1,311 @@
+# Common data types
+
+## Processed, coregistered and/or resampled volumes
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
```

Comment:
Regarding 
https://github.com/bids-standard/bids-specification/pull/109#issuecomment-46359249 8I think this is where other modalities can be mentioned as well. In general I would say that it could be 

```<pipeline_name>/
     sub-<participant_label>/
         func|anat|dwi|eeg|meg|ieeg|whatever_future_modality/```

Reply:
This is would be the place to add other modalities, but I would prefer to add them one by one with subsequent PRs to keep the scope of this work more manageable.